firefox-ios/Client/Frontend/Home/Homepage Rebuild/Layout/HomepageSectionLayoutProvider.swift

```diff
@@ -128,6 +191,8 @@ final class HomepageSectionLayoutProvider: FeatureFlaggable {
 
     private var logger: Logger
     private var windowUUID: WindowUUID
+    private var measurementsCache = LayoutMeasurementCache()
+    private var headerHeightCache: [HeaderMeasurementKey: CGFloat] = [:]
```

Comment:
We can move this `headerHeightCache` into under the `HomepageMeasumerCache` and in its own file instead, we can retrieve using `measurementsCache.getHeaderHeight`